lua/scratch/scratch_file.lua

```diff
@@ -16,22 +15,22 @@ local function write_lines_to_buffer(lines)
 end
 
 local function hasDefaultContent(ft)
-  local config_data = config.getConfig()
+  local config_data = config.get_and_update_config()
```

Comment:
I think it makes sense to just use `vim.g.scratch_config` directly. Here and afterwards.

Reply:
Yeah, initially I use just vim.g.scratch_config, but then I thought what if user changed their config in an unexpected way, I mean directly to their config json during runtime

Reply:
Then they should call that function themselves)
If you want to make it easier for the user, then expose it as a vim command. Something like `:ScractchReloadJson`.